TriviaServer/TriviaServer.DAL/Repositories/GameRepository.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TriviaServer.Models.Repositories;
+
+namespace TriviaServer.DAL.Repositories
+{
+    class GameRepository : IGameRepository
+    {
+        private readonly TriviaRankContext _context;
+        public GameRepository(TriviaRankContext context)
+        {
+            _context = context;
+        }
+        public async Task<Models.GameModel> CreateGame(int ownerId, string gameName, int totalQuestions, bool isPublic)
+        {
+            Game createGame = await _context.Games
```

Comment:
Is this being used anywhere?

Reply:
My understanding was that you have to query in order to add/save changes. if this isn't the case, I will modify the code.

Reply:
What I'm trying to do is add a game to the DB to create a game ID, then pass that back to the user.